### PR TITLE
Create recipe for pymedphys

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

File: recipes/pymedphys/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+test:
+  imports:
+    - pymedphys
+    - pymedphys.level1
+    - pymedphys.level2
+
+about:
+  home: The package home page
+  license: GNU Affero General Public v3 or later (AGPLv3+)
```
Comment (mariusvniekerk) [inline recipes/pymedphys/meta.yaml]: Change this to AGPL-3.0+

File: recipes/pymedphys/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - pydicom
+
+test:
+  imports:
+    - pymedphys
+    - pymedphys.level1
+    - pymedphys.level2
+
+about:
+  home: The package home page
```
Comment (mariusvniekerk) [inline recipes/pymedphys/meta.yaml]: Can you add the home page in here?

Comment (SimonBiggs) [reply recipes/pymedphys/meta.yaml]: Certainly, just the GitHub repo address?

File: recipes/pymedphys/meta.yaml (hunk 1 above)
Comment (SimonBiggs) [reply recipes/pymedphys/meta.yaml]: Done. With this by the way, that text came straight from using the conda skeleton recipe. Might be worth changing the text upstream also?

**Comment by SimonBiggs** [comment]
Also, are there ways to make the meta.yaml file DRY with respect to the setup.py within the repo? For example the license file, the dependencies, the imports, etc. The way I have set it up there is a lot of duplication. Would I be able to make the meta.yaml extract that info from the setup.py dynamically?

**Comment by mariusvniekerk** [comment]
Closest to that is probably tools like conda-skeleton that generate them.  That being said, once they are made neither setup.py nor meta.yaml change all that much.  
Welcome to conda-forge!

**Pull request merged**